tools/clang/test/HLSL/shader_cc_mismatch.hlsl

```diff
@@ -0,0 +1,8 @@
+// RUN: %clang_cc1 -fsyntax-only -ffreestanding -verify %s
+
+
+[shader("compute")] 
+[shader("compute")] 
+[ numthreads( 64, 2, 2 ) ]  /* expected-no-diagnostics */
```

Comment:
Can we avoid adding so many separate test files?  Part of the point of the verifier tests is that you can verify a lot of related scenarios in one file, rather than having to have a whole bunch of files, one testing each failure case.  You should be able to put all of these into one `shader_attribute.hlsl` verifier test for testing all the shader attribute diagnostics.

Reply:
Alternatively since @python3kgae just deleted the old test running logic, we could make these lit tests using the DXC -verify flag. That eliminates the need for any unit test cases.

Reply:
I merged the files into two new files. One for expected errors/notes, and another that expects no errors/notes.